dockerfiles/build_manylinux_x86_64.Dockerfile

```diff
@@ -95,22 +104,19 @@ RUN which gcc && gcc --version && \
 # in git 2.35.3
 RUN git config --global --add safe.directory '*'
 
-######## vcpkg ########
-# vcpkg is used to install OpenMPI and can be dropped once the latter
-# is vendored into TheRock.
-WORKDIR /opt
-ENV VCPKG_HASH="3f5ad7be7693ce6ac5599ddb7cc24f260b9d44f9"
-COPY install_vcpkg.sh ./
-RUN yum install -y zip unzip
-RUN ./install_vcpkg.sh "${VCPKG_HASH}"
-
-######## OpenMPI ########
+######## Open MPI 5.0.8 (from source) ########
 # OpenMPI is currently not vendored into TheRock and temorarily installed
-# via vcpkg: https://github.com/ROCm/TheRock/issues/1284
-RUN /opt/vcpkg/vcpkg install openmpi:x64-linux
-ENV PATH="/opt/vcpkg/installed/x64-linux/tools/openmpi/bin:${PATH}"
-ENV PKG_CONFIG_PATH="/opt/vcpkg/installed/x64-linux/lib/pkgconfig:${PKG_CONFIG_PATH:-}"
-ENV LD_LIBRARY_PATH="/opt/vcpkg/installed/x64-linux/lib:${LD_LIBRARY_PATH}"
+ENV OMPI_VER=5.0.8
+ENV OMPI_PREFIX=/opt/openmpi-${OMPI_VER}
+
+# Copy and execute the build/install portion
+COPY install_openmpi.sh /tmp/install_openmpi.sh
+RUN chmod +x /tmp/install_openmpi.sh && /tmp/install_openmpi.sh && rm -f /tmp/install_openmpi.sh
```

Comment:
You can rather set file permissions before adding this to git rather than needing to call `chmod` here.